pkgs/development/python-modules/libclang/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, llvmPackages
+, setuptools
+, writeText
+}:
+
+let
+  libclang = llvmPackages.libclang;
+
+  pyproject_toml = writeText "pyproject.toml" ''
+    [build-system]
+    requires = ["setuptools>=42", "wheel"]
+    build-backend = "setuptools.build_meta"
+  '';
+
+  setup_cfg = writeText "setup.cfg" ''
+    [metadata]
+    name = clang
+    version = ${libclang.version}
+
+    [options]
+    packages = clang
+  '';
+in buildPythonPackage {
+  pname = "libclang";
+  format = "pyproject";
+
+  inherit (libclang) version src;
+
+  buildInputs = [ setuptools ];
+
+  postUnpack = ''
+    # set source root to python bindings
+    sourceRoot="$sourceRoot/clang/bindings/python"
```

Comment:
sourceRoot="$sourceRoot/bindings/python"

Reply:
Sorry, I was still using LLVM 11 apparently, so this is unnecessary.

Reply:
Hmm we won't be using llvm 11 anymore after staging-next is merged, but it looks like at least 12 is still lile that too. I'll adjust it later to support both possible `sourceRoot`s just in case someone wants it

Thank you